tests/baselines/reference/declarationEmit_bindingPatters.js

```diff
@@ -0,0 +1,26 @@
+//// [declarationEmit_bindingPatters.ts]
+
+const k = ({x: z = 'y'}) => { }
+
+var a;
+function f({} = a, [] = a, { p: {} = a} = a) {
+}
+
+//// [declarationEmit_bindingPatters.js]
+var k = function (_a) {
+    var _b = _a.x, z = _b === void 0 ? 'y' : _b;
+};
+var a;
+function f(_a, _b, _c) {
+    var _a = a;
+    var _b = a;
+    var _d = (_c === void 0 ? a : _c).p, _e = _d === void 0 ? a : _d;
+}
+
+
+//// [declarationEmit_bindingPatters.d.ts]
+declare const k: ({x:z}: {

```

Comment:
The formatter actually doesn't seem to touch curlies in a binding pattern, and this is how we usually write code in our compiler:

```ts
const { exportsWithDuplicate } = lookupTable[id];
```

So I'd argue that the formatter actually needs to be updated.